chef/data_bags/crowbar/template-monasca.json

```diff
@@ -38,12 +38,18 @@
           "service_role": "monasca-agent"
         }
       },
-      "api" : {
-        "bind_host" : "*",
-        "bind_port" : 8070,
-        "processes" : 3,
-        "ssl" : false,
-        "threads" : 10
+      "api": {
+        "bind_host": "*",
+        "bind_port": 8070,
+        "processes": 3,
+        "ssl": false,
+        "threads": 10,
+        "log_level": "INFO"
+      },
+      "log_api": {
+        "port": 5607,
+        "ssl": false,
```

Comment:
we might want to do that as a sub dict with the usual settings going forward. for now I would not want to worry about it